react/components/ProductImages/components/ProductImage.tsx

```diff
@@ -18,6 +18,11 @@ interface Props {
 
 type AspectRatio = string | number
 
+const CSS_HANDLES = [
+  'productImage',
+  'productImageImg'
```

Comment:
kkkk
maybe this would sound better:
```suggestion
  'productImageTag'
```